dnf-behave-tests/features/plugins-core/debuginfo-install.feature

```diff
@@ -2,7 +2,7 @@ Feature: Tests for debuginfo symbols
 
 @bz1585137
 Scenario: debuginfo-install reports an error when debuginfo is not found
-Given I use the repository "dnf-ci-fedora"
+Given I use the repository "debuginfo-install"
```

Comment:
Well.. how can it happen that the repositories get mixed up with your system repositories?

Also, the prefix itself (`dnf-ci-`) doesn't say it's a repository. It's just that you know the repositories have this naming convention and nothing else does.

In the context of tests, I find it not useful and redundant. In case we do need to distinguish these from system repositories, then we should probably use it. I just didn't find it necessary.

But I'd rather it was a suffix (so that it's easier to read and work with in directory structure) and maybe it could just be `-ci`?

I've used this as a proposition to change the convention, if we agree on a change we should gradually migrate, otherwise I'll change this to the current convention... We should discuss with @mblaha and others...